bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -6891,6 +6891,9 @@ public synchronized void close() {
 	/**
 	 * Store the known status for the FsContent in the database Note: will not
 	 * update status if content is already 'Known Bad'
+	 * 
+	 * This method is no longer called during normal ingest, as we save writing
```

Comment:
These comments probably don't belong at the SleuthKit level, where we have no concept of ingest, since that is an Autopsy concept.